src/data/matrix/basic.lean

```diff
@@ -742,6 +750,14 @@ lemma mul_vec_smul_assoc (A : matrix m n α) (b : n → α) (a : α) :
   A.mul_vec (a • b) = a • (A.mul_vec b) :=
 by { ext, apply dot_product_smul }
 
+lemma vec_mul_to_mul_vec (A : matrix m n α) (x : m → α) :
+  vec_mul x A = mul_vec Aᵀ x :=
```

Comment:
to match the lemma below:
```suggestion
lemma mul_vec_transpose (A : matrix m n α) (x : m → α) :
  mul_vec Aᵀ x = vec_mul x A :=
```

Reply:
Oh, I am so sorry, I didn't see that comment. Will change right away